examples/example_001.py

```diff
@@ -1,82 +1,151 @@
 # -*- coding: utf-8 -*-
 """
-Dummy example
-
-@author: asaez
+Python Flight Mechanics Engine (PyFME).
+Copyright (c) AeroPython Development Team.
+Distributed under the terms of the MIT License.
+
+Example with trimmed aircraft.
+The main purpose of this example is to check if the aircraft trimmed in a given
+state maintains the trimmed flight condition.
+Trimmed in stationary, horizontal, symmetric, wings level flight.
 """
 
 import numpy as np
 import matplotlib.pyplot as plt
 
+from pyfme.aircrafts import cessna_310
 from pyfme.models.system import System
+from pyfme.utils.trimmer import steady_state_flight_trim
+from pyfme.utils.anemometry import calculate_alpha_beta_TAS
+from pyfme.environment.isa import atm
 
 
+if __name__ == '__main__':
 
-def forces():
-    return np.array([0, 0, -9.8*500])
-
-
-def moments():
-    return np.array([0, 0, 0])
-
+    # Aircraft parameters.
+    mass, inertia = cessna_310.mass_and_inertial_data()
 
-if __name__ == '__main__':
+    # Initial conditions.
+    TAS_ = 312 * 0.3048  # m/s
+    h = 8000 * 0.3048  # m
+    psi_0 = 3  # rad
+    x_0, y_0 = 0, 0  # m
 
-    # Case params
-    mass = 500
-    inertia = np.array([[1, 0, 0],
-                        [0, 1, 0],
-                        [0, 0, 1]])
+    # Trimming.
+    trim_results = steady_state_flight_trim(cessna_310, h, TAS_, gamma=0,
+                                            turn_rate=0)
 
-    # initial conditions
-    u, v, w = 0, 0, 0
-    p, q, r = 0, 0, 0
+    lin_vel, ang_vel, theta, phi, alpha_, beta_, control_vector = trim_results
 
-    theta, phi, psi = 0, 0, 0
-    x, y, z = 0, 0, 5000
+    # Time.
+    t0 = 0  # s
+    tf = 30  # s
+    dt = 1e-2  # s
 
-    t0 = 0
-    tf = 10
-    dt = 1e-2
     time = np.arange(t0, tf, dt)
 
+    # Results initialization.
     results = np.zeros([time.size, 12])
 
-    velocities = np.empty([time.size, 6])
-    results[0, 0:6] = u, v, w, p, q, r
-    results[0, 6:9] = theta, phi, psi
-    results[0, 9:12] = x, y, z
+    results[0, 0:3] = lin_vel
```

Comment:
The alternative would be to return a too large number of variables I guess, so I have no other suggestion.